tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/ChunkMetadata.java

```diff
@@ -224,16 +228,12 @@ public boolean equals(Object o) {
     ChunkMetadata that = (ChunkMetadata) o;
     return offsetOfChunkHeader == that.offsetOfChunkHeader
         && version == that.version
-        && Objects.equals(measurementUid, that.measurementUid)
-        && tsDataType == that.tsDataType
-        && Objects.equals(deleteIntervalList, that.deleteIntervalList)
-        && Objects.equals(statistics, that.statistics);
+        && partialFilePathForCache.equals(that.partialFilePathForCache);
```

Comment:
hi,  equals and hashcode can not only use for cache,  ChunkMetadata have other place for judgments. 
test case have some fail.